services/checks/checks.go

```diff
@@ -110,12 +115,22 @@ func (s *Service) Run(ctx context.Context) {
 	defer ticker.Stop()
 
 	for {
-		// FIXME do that only if STT is enabled in settings https://jira.percona.com/browse/SAAS-30
-		if true {
+		var sttEnabled bool
```

Comment:
Talked with Roma – yes, we should just delay the first check for one minute.

Reply:
Timeout added